contracts.proto

```diff
@@ -37,6 +38,8 @@ service NativeAutomator {
   rpc handlePermissionDialog(HandlePermissionRequest) returns (Empty) {}
   rpc setLocationAccuracy(SetLocationAccuracyRequest) returns (Empty) {}
   rpc tapOnNotification(TapOnNotificationRequest) returns (Empty) {}
+
+  rpc debug(Empty) returns (Empty) {}
```

Comment:
It's really useful for implementing some one-off debugging functionality. I want this to stay.

Reply:
Imo this is absolutely fine